config/ros_control_controllers.yaml

```diff
@@ -0,0 +1,210 @@
+    - /left/j1
+    - /left/j2
+    - /left/j3
+    - /left/j4
+    - /left/j5
+    - /left/j6
+    - /left/j7
+    - /right/j1
+    - /right/j2
+    - /right/j3
+    - /right/j4
+    - /right/j5
+    - /right/j6
+    - /right/j7
+  gains:
+    left/j1:  {p: 900.0,  i: 2.5, d: 10.0, i_clamp_min: -1.0, i_clamp_max: 1.0}
+    left/j2:  {p: 2500.0, i: 5.0, d: 20.0, i_clamp_min: -1.0, i_clamp_max: 1.0}
+    left/j3:  {p: 600.0,  i: 2.5, d: 10.0, i_clamp_min: -1.0, i_clamp_max: 1.0}
+    left/j4:  {p: 500.0,  i: 0.5, d: 2.5,  i_clamp_min: -1.0, i_clamp_max: 1.0}
+    left/j5:  {p: 40.0,   i: 0.5, d: 0.5,  i_clamp_min: -1.0, i_clamp_max: 1.0}
+    left/j6:  {p: 40.0,   i: 0.5, d: 0.5,  i_clamp_min: -1.0, i_clamp_max: 1.0}
+    left/j7:  {p: 12.0,   i: 0.5, d: 0.05, i_clamp_min: -1.0, i_clamp_max: 1.0}
+    right/j1: {p: 900.0,  i: 2.5, d: 10.0, i_clamp_min: -1.0, i_clamp_max: 1.0}
+    right/j2: {p: 2500.0, i: 5.0, d: 20.0, i_clamp_min: -1.0, i_clamp_max: 1.0}
+    right/j3: {p: 600.0,  i: 2.5, d: 10.0, i_clamp_min: -1.0, i_clamp_max: 1.0}
+    right/j4: {p: 500.0,  i: 0.5, d: 2.5,  i_clamp_min: -1.0, i_clamp_max: 1.0}
+    right/j5: {p: 40.0,   i: 0.5, d: 0.5,  i_clamp_min: -1.0, i_clamp_max: 1.0}
+    right/j6: {p: 40.0,   i: 0.5, d: 0.5,  i_clamp_min: -1.0, i_clamp_max: 1.0}
+    right/j7: {p: 12.0,   i: 0.5, d: 0.05, i_clamp_min: -1.0, i_clamp_max: 1.0}
+  constraints: # TODO what are these values?
+    goal_time: 0.6
+    stopped_velocity_tolerance: 0.02
+    left/j1:  {trajectory: 0.05, goal: 0.02}
+    left/j2:  {trajectory: 0.05, goal: 0.02}
+    left/j3:  {trajectory: 0.05, goal: 0.02}
+    left/j4:  {trajectory: 0.05, goal: 0.02}
+    left/j5:  {trajectory: 0.05, goal: 0.02}
+    left/j6:  {trajectory: 0.05, goal: 0.02}
+    left/j7:  {trajectory: 0.05, goal: 0.02}
+    right/j1: {trajectory: 0.05, goal: 0.02}
+    right/j2: {trajectory: 0.05, goal: 0.02}
+    right/j3: {trajectory: 0.05, goal: 0.02}
+    right/j4: {trajectory: 0.05, goal: 0.02}
+    right/j5: {trajectory: 0.05, goal: 0.02}
+    right/j6: {trajectory: 0.05, goal: 0.02}
+    right/j7: {trajectory: 0.05, goal: 0.02}
+  stop_trajectory_duration: 0.5
+  state_publish_rate: 25
+
+
+# Hand controllers -------------------------------------------------------------
+right_hand_controller:
+  type: position_command_controller/PositionCommandController
+  resource_name: /right
+
+left_hand_controller:
+  type: position_command_controller/PositionCommandController
+  resource_name: /left
+
+
+# Force/Torque Sensor Controller -----------------------------------------------
+force_torque_sensor_controller:
+  type: force_torque_sensor_controller/ForceTorqueSensorController
+  publish_rate: 50
+  
+
+# Tare controllers -------------------------------------------------------------
+right_tare_controller:
+  type: trigger_controller/TriggerController
+  resource_name: /right/ft_tare
+
+left_tare_controller:
+  type: trigger_controller/TriggerController
+  resource_name: /left/ft_tare
```

Comment:
I am concerned about the number of times we have to copy/paste PID gains in this file. Could we use something like [anchors and aliases](http://www.yaml.org/spec/1.2/spec.html#anchor//) to DRY?